tests/scripts/run-xunit-perf.cmd

```diff
@@ -96,21 +96,40 @@ setlocal
   rem CORE_ROOT environment variable is used by some benchmarks such as Roslyn / CscBench.
   set CORE_ROOT=%LV_SANDBOX_DIR%
   set LV_RUNID=Perf-%ETW_COLLECTION%
-  set BENCHNAME_LOG_FILE_NAME=%LV_BENCHMARKS_OUTPUT_DIR%\%LV_RUNID%-%BENCHNAME%.log
 
+  set "LV_BENCHMARK_OUTPUT_DIR="
```

Comment:
I'm confused by this part.  Why do we set this to something on line 13 above, and then unset it here?

Reply:
Agreed this seems weird.

Reply:
Oh, it's a different variable. LV_SANDBOX on line 13, LV_BENCHMARK here.